apps/protocol-frontend/src/components/GlobalFilter.tsx

```diff
@@ -37,7 +37,7 @@ const GlobalFilter = ({
         </InputLeftElement>
         <Input
           placeholder={`Search ${rowCount} ${
-            rowCount > 1 ? 'Contributions' : 'Contribution'
+            rowCount === 1 ? 'Contribution' : 'Contributions'
```

Comment:
**Not so important**: 
I am starting to notice this frequent plural pattern; I think we can make a function: `plural(countable, count)` or `pluralize(countable, count)` to be more clear & readable.